server/schedule/filters.go

```diff
@@ -412,3 +412,60 @@ func (f StoreStateFilter) filterMoveRegion(opt Options, store *core.StoreInfo) b
 	}
 	return false
 }
+
+// BlacklistType the type of BlackListStore Filter.
+type BlacklistType int
+
+// some flags about blacklist type.
+const (
+	SignSource BlacklistType = iota + 1
+	SignTarget
+	SignSourceAndTarget
```

Comment:
When do we use it?

Reply:
We do not used it now.

Reply:
how about using bit flags?
```
source = 1 << iota
target
```
Then `if flag&source` or `if flag&target`